packages/plugins/rudder_plugin_web/lib/internal/web_js.dart

```diff
@@ -1,5 +1,5 @@
 @JS()
-library rudder_analytics.js;
+library;
```

Comment:
Can we move these code changes to a separate PR?

I think we should keep this one focused solely on updating the dependencies.

Let me know your thoughts.

Reply:
For now, I would propose to revert the current changes.

Reply:
Actually, they came free as part of lint and formatting changes in the pre commit hook. The changes are only in a couple of files and are harmless too.

Do you sense any breaking changes?

Reply:
Nope. It's fine then.

(I thought we could make the changes directly in the main PR, if needed.)